working-together/README.md

```diff
@@ -0,0 +1,23 @@
+Working Together
+==============
+
+A guide for working well together.
+
+
+Pair Programming
+-------
+
+**Before:**
+* Agree Scope (Pairing for two hours? Until the ticket is complete? Just get pass the bug?)
+* Agree Working Environment (Two keyboards? VIM/what other tools?)
+* Agree Pairing Style (Time-based? Ping pong? To the next traffic light?)
+
+**During:**
+* Keep the chat going (Think aloud. Encourage/support)
+* Keep switching (Follow the pairing style)
+* Keep both involved ("Could we do this another way?")
+* Keep breaking (Have a break, have a…)
+* Keep checking in ("Could we google this separately?")
```

Comment:
Unlike thoughtbot, Google capitalizes the first letter of their name.

But since a loud group of us are on DDG, how about something like "Could we search for a guide separately?"?